sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -522,7 +523,7 @@ case class DescribeTableCommand(table: TableIdentifier, isExtended: Boolean, isF
 
   private def describeSchema(schema: Seq[CatalogColumn], buffer: ArrayBuffer[Row]): Unit = {
     schema.foreach { column =>
-      append(buffer, column.name, column.dataType.toLowerCase, column.comment.orNull)
+      append(buffer, column.name, column.dataType.toLowerCase, column.comment.getOrElse(""))
```

Comment:
Yea. If it is null, let's keep it as null. Changing a null to an empty string actually destroys the information.